src/batch-client/index.ts

```diff
@@ -476,6 +497,16 @@ export class ZimbraBatchClient {
 			)
 		);
 
+	public singleRequest = (options: JsonRequestOptions) => {
+		singleJSONRequest({
```

Comment:
by calling this function directly we are skipping code for notifications handling